public/app.js

```diff
@@ -11,7 +11,7 @@ import route from 'can/route/';
 import Session from './models/session';
 import "can/route/pushstate/";
 import stache from "can/view/stache/";
-import "./util/prefilter";
```

Comment:
It is replaced by a general util import. Shouldn't a stache helper be imported globally unless it's just being registered for one component (in which case it belongs in the helpers object)?